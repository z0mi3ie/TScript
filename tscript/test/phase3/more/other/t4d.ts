//print = function (x) { console.log(x); };

//readln = function () {
//  var cnt = 0;
//  return function () {
//    if (cnt < 5) return "line " + ++cnt + "\n";
//    else return "";
//  };
//}();

var x;

x = readln();
while (x)
{
  print (x);
  x = readln();
}

print (x);

