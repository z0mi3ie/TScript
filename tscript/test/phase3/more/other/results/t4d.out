when the going gets weird

the weird

turn pro




